Common/src/main/java/me/egg82/tfaplus/services/InternalAPI.java

```diff
@@ -762,32 +778,36 @@ public static void setLogin(UUID uuid, String ip, long ipTime, JedisPool redisPo
         }
 
         // Redis
-        Redis.update(result, ipTime, redisPool, redisConfigNode);
+        Redis.update(result, ipTime, ConfigUtil.getRedisPool(), ConfigUtil.getRedisConfigNode());
 
         // RabbitMQ
-        RabbitMQ.broadcast(result, ipTime, rabbitConnection);
+        try {
+            RabbitMQ.broadcast(result, ipTime, getRabbitConnection());
```

Comment:
Not using the broadcast method, here?
Redis & Rabbit broadcast/update should probably be in one method, if we're going to make it a method anyway.

Reply:
I see the broadcast method here?